### PR TITLE
Fixed 122 LGTM Issues

### PR DESCRIPTION
**Pull request by aaron-russell**
#### Tests
Does this PR include tests for the changes introduced?
- [ ] Yes
- [X] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: server/aws/kinesis.js [hunk 1]
```diff
@@ -7,13 +7,13 @@ let kinesis = null;
...
     apiVersion: '2013-12-02',
-    region
+    region,
   });
 };
 
 const ACTIONS = ['created', 'updated', 'deleted'];
 
-const establishmentPump = async (action, establishment)  => {
+const establishmentPump = async (action, establishment) => {
```
Comment (chadtomkiss-madetech) [inline server/aws/kinesis.js:16]: Did the Kinesis stuff get removed?

File: server/routes/login.js [hunk 2]
```diff
@@ -19,7 +17,7 @@ const sendMail = require('../utils/email/notify-email').sendPasswordReset;
 
 const tribalHashCompare = (password, salt, expectedHash) => {
```
Comment (chadtomkiss-madetech) [inline server/routes/login.js:18]:
```
// if this found login account is a migrated tribal account, and there is no current hash, then
//  we need to first validate password using tribal hashing
```

There is a process to migrate hashes when logging in.. do we know if there are any logins that have yet to be migrated? If not, we can probably remove this code completely.

File: server/aws/kinesis.js (hunk 1 above)
Comment (aaron-russell) [reply server/aws/kinesis.js:16]: I think so actually, I can remove this file, not a problem

File: server/routes/login.js (hunk 2 above)
Comment (aaron-russell) [reply server/routes/login.js:18]: I'm not sure about that, I would guess so, think we will know once we start deleting old accounts

File: server/sendEmail.js [hunk 3]
```diff
@@ -1,93 +1,94 @@
 // this is simply to test sending emails via AWA SES
```
Comment (chadtomkiss-madetech) [inline server/sendEmail.js:1]: Can this file be deleted? It's tied to Warrens email etc.

File: server/routes/login.js (hunk 2 above)
Comment (aaron-russell) [reply server/routes/login.js:18]: Think we're best waiting for this one

**Review by chadtomkiss-madetech — APPROVED** [review]

**Pull request merged**